cognite/neat/_rules/transformers/_converters.py

```diff
@@ -157,73 +149,94 @@ def _fix_properties(cls, definitions: SheetList[InformationProperty]) -> SheetLi
         return fixed_definitions
 
 
-class PrefixEntities(RulesTransformer[ReadRules[T_InputRules], ReadRules[T_InputRules]]):  # type: ignore[type-var]
-    """Prefixes all entities with a given prefix."""
+class PrefixEntities(ConversionTransformer):  # type: ignore[type-var]
+    """Prefixes all entities with a given prefix if they are in the same space as data model."""
 
     def __init__(self, prefix: str) -> None:
         self._prefix = prefix
 
     @property
     def description(self) -> str:
-        return f"Prefixes all views with {self._prefix!r}"
+        return f"Prefixes all entities with {self._prefix!r} prefix if they are in the same space as data model."
 
-    def transform(self, rules: ReadRules[T_InputRules]) -> ReadRules[T_InputRules]:
-        in_ = rules.rules
-        if in_ is None:
-            return rules
-        copy: T_InputRules = dataclasses.replace(in_)
-        if isinstance(copy, InformationInputRules):
-            prefixed_by_class: dict[str, str] = {}
+    def transform(self, rules: InformationRules | DMSRules) -> InformationRules | DMSRules:
```

Comment:
You will either do information->Infomrtion, or DMS->DMS
```suggestion
    def transform(self, rules: T_VerifiedIn) -> T_VerifiedIn:
```

Maybe an even better alternative use overload

Reply:
Didn't I set up overload correctly ? hm ... that is strange